data/brands/amenity/fast_food.json

```diff
@@ -7439,7 +7508,13 @@
           "om",
           "qa",
           "sa",
-          "ye"
+          "eg",
```

Comment:
Looks like Mcdonalds' global entry haven't exclude other countries

It's a good time to do that

https://github.com/osmlab/name-suggestion-index/blob/f51f320c3d23441d43452927bd6cc669edf005a5/data/brands/amenity/fast_food.json#L3822-L3827

Reply:
I think no more (McDonald's) in Russia

Reply:
what about other non Latin countries such as china and japan? it's not excluded global

Reply:
Local brands probably should be excluded from the global item (even though this index has not consistently done it, we should).

The iD validator will still pick the most local one when matching features to their likely brand, but excluding them from the global item will prevent it from showing up in the preset list, which is probably what people want.

Reply:
OK!
understood
thanks